packages/cli/tests/e2e/apiconnection/verifyScaffold.tests.ts

```diff
@@ -83,13 +82,12 @@ describe("Add Api Connection Tests", function () {
     const pkgFile = await fs.readJson(path.join(projectPath, "bot", "package.json"));
     const deps = pkgFile.dependencies;
     chai.assert.exists(deps["@microsoft/teamsfx"]);
-    chai.assert.exists(deps["@microsoft/teams-js"]);
```

Comment:
why delete the teams-js

Reply:
chatted with Ning, teams-js is only used in frontend code as peer dependency. since we target nodejs project, we do not need to install it.